Features/ModerationCommands/ModCommandGroup.cs

```diff
@@ -0,0 +1,20 @@
+using DSharpPlus.Commands;
+using DSharpPlus.Commands.ContextChecks;
+using DSharpPlus.Entities;
+
+namespace ArtcordAdminBot.Features.ModerationCommands
+{
+    [Command("moderation")]
```

Comment:
Could be shortened down to "mod", especially since these are commands that will get used a lot